client.go

```diff
@@ -72,9 +72,7 @@ type Marathon interface {
 	// get a list of tasks for a specific application
 	Tasks(application string) (*Tasks, error)
 	// get a list of all tasks
-	AllTasks() (*Tasks, error)
-	// get a listing of the task ids
-	ListTasks() ([]string, error)
+	AllTasks(status string) (*Tasks, error)
```

Comment:
I agree with url.Values. The PR seems good for the rest and it's a very useful feature.